### ISSUE TITLE
Feature Request

### ISSUE THREAD

**Issue by begroff** [deadlydog/Set-ProjectFilesClickOnceVersion#1]
Have you ever thought of updating this to change additional nodes in the property group for ClickOnce like PublishURL and InstallURL?

I have a need for it at my company as we deploy ClickOnce apps in different locations in Dev and Test/Prod and I want to add this script to their associated builds. I made some changes where I add those two parameters and set/update them before the logic of getting setting app version. 

I forked the repository and was going to submit a pull request after adding some more tests. 

Thoughts?

**Comment by deadlydog** [comment]
Sounds like a nice feature to have. I'd definitely take a look if you create a PR for it 👍

**Comment by begroff** [comment]
Cool, I have a question, what is the %2a signify in line of code below. When I run it locally (i.e. not on the build server) I see %2a in the actual file.

```ps
Set-XmlNodesElementTextValue -xml $xml -node $clickOncePropertyGroup -elementName 'ApplicationVersion' -textValue "$newMajorMinorBuild.%2a"
```

**Comment by deadlydog** [comment]
The `%2a` is automatically added to the .csproj file when you convert your app to a ClickOnce app via Visual Studio. e.g. it adds the following fields to the xml file:

```xml
    <ApplicationRevision>52</ApplicationRevision>
    <ApplicationVersion>1.0.1.%2a</ApplicationVersion>
```

The `%2a` signifies that it should be replaced with the `ApplicationRevision` value, which is auto-incremented by Visual Studio every time you publish from VS. The `ApplicationVersion` will be the ClickOnce version of your application.

This is something that's built into VS/MsBuild, not something custom that I created.

**Comment by begroff** [comment]
Excellent thanks for clarifying. Our ClickOnce apps have this custom splash screen that displays the assembly version on startup. Any ideas on how I would be able to match the Click Once version from the build to the assembly version? My initial thought was to adjust your script further with some logic to update the Assembly.cs file in the build but I’m not sure how that would affect source control.

**Comment by deadlydog** [comment]
Yeah, I talk briefly about a few of the different ways to handle the assembly version number vs. the clickonce version number [in one of my blog posts](http://blog.danskingdom.com/continuously-deploy-your-clickonce-application-from-your-build-server/). There may be other options though.

**Comment by deadlydog** [comment]
Thanks so much for making PR #2 which adds the `PublishUrl` and `InstallUrl` parameters to the script 👍 Closing this guy out now since that functionality has been merged in.

**Comment by deadlydog** [comment]
This functionality is now included in v2.1.0 of the script :)
